Chat/package.json

```diff
@@ -17,20 +17,21 @@
     "lint:quiet": "eslint */**/*.{ts,tsx} --quiet"
   },
   "dependencies": {
+    "@azure/communication-calling": "1.4.4",
```

Comment:
nit: chat sample shouldn't need the calling dep

Reply:
Unfortunately we do. You'll get a warning about missing peerDependecy if you don't include this.

Reply:
Oh yeah that's right